oraichain/assetlist.json

```diff
@@ -50,6 +50,516 @@
+      },
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/injective/images/inj.png",
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/injective/images/inj.svg"
+        }
+      ]
+    },
+    {
+      "description": "The USDT token is the cw20 token for the Oraichain.",
+      "denom_units": [
+        {
+          "denom": "cw20:orai12hzjxfh77wl572gdzct2fxv2arxcwh6gykc7qh",
+          "exponent": 0
+        },
+        {
+          "denom": "USDT",
+          "exponent": 6
+        }
+      ],
+      "type_asset": "cw20",
+      "address": "orai12hzjxfh77wl572gdzct2fxv2arxcwh6gykc7qh",
+      "base": "cw20:orai12hzjxfh77wl572gdzct2fxv2arxcwh6gykc7qh",
+      "name": "Usdt",
+      "display": "usdt",
+      "symbol": "USDT",
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/axelar/images/usdt.png",
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/axelar/images/usdt.svg"
+        }
+      ],
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/axelar/images/usdt.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/axelar/images/usdt.svg"
+      }
+    },
+    {
+      "description": "The AIRI token is the cw20 token for the Oraichain.",
+      "denom_units": [
+        {
+          "denom": "cw20:orai10ldgzued6zjp0mkqwsv2mux3ml50l97c74x8sg",
+          "exponent": 0
+        },
+        {
+          "denom": "AIRI",
+          "exponent": 6
+        }
+      ],
+      "type_asset": "cw20",
+      "address": "orai10ldgzued6zjp0mkqwsv2mux3ml50l97c74x8sg",
+      "base": "cw20:orai10ldgzued6zjp0mkqwsv2mux3ml50l97c74x8sg",
+      "name": "Airight",
+      "display": "airi",
+      "coingecko_id": "airight",
+      "symbol": "AIRI",
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/oraichain/images/airi.png",
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/oraichain/images/airi.svg"
+        }
+      ],
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/oraichain/images/airi.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/oraichain/images/airi.svg"
+      }
+    },
+    {
+      "description": "The ORAIX token is the cw20 token for the Oraichain.",
+      "denom_units": [
+        {
+          "denom": "cw20:orai1lus0f0rhx8s03gdllx2n6vhkmf0536dv57wfge",
+          "exponent": 0
+        },
+        {
+          "denom": "ORAIX",
+          "exponent": 6
+        }
+      ],
+      "type_asset": "cw20",
+      "address": "orai1lus0f0rhx8s03gdllx2n6vhkmf0536dv57wfge",
+      "base": "cw20:orai1lus0f0rhx8s03gdllx2n6vhkmf0536dv57wfge",
+      "name": "Oraix",
+      "display": "oraix",
```

Comment:
Most of the display denoms specified in the "display" property are not defined amongst the denom_units, but it must be.